data/templates/post-install-photon.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+<%_ if (undefined !== networkDevices) { _%>
```

Comment:
Use `typeof networkDevies !== 'undefined'` rather than yours as the networkDevices is optional value, it it doesn't exist, the ejs will throw error that it cannot find the reference of networkDevices.